pyproject.toml

```diff
@@ -61,38 +61,44 @@ tests = [
     "scipy==1.10.1; python_version < '3.9'", # to support python 3.8
     "scipy==1.11.1; python_version > '3.8'",
     "pandas==2.0.3",
+    "pyiges==0.3.1",
+    "geomdl==5.3.1",
     "pytest==7.4.0",
     "pytest-cov==4.1.0",
-    "pyvista==0.39.1",
+    "pyvista==0.40.0",
     "pyansys-tools-report==0.6.0",
     "vtk==9.2.6",
     "pytest-rerunfailures==11.1.2",
 ]
 doc = [
     "sphinx==6.2.1",
     "ansys-dpf-core==0.8.1",
-    "ansys-mapdl-reader==0.52.16",
+    "ansys-mapdl-reader==0.52.17",
     "ansys-sphinx-theme==0.9.9",
     "grpcio==1.51.1",
     "imageio-ffmpeg==0.4.8",
     "imageio==2.31.1",
     "jupyter_sphinx==0.4.0",
     "jupyterlab>=3.2.8",
     "matplotlib==3.7.1",
+    "nest-asyncio==1.5.6",
     "numpydoc==1.5.0",
     "pandas==2.0.3",
     "plotly==5.15.0",
+    "pyiges==0.3.1",
+    "geomdl==5.3.1",
     "pypandoc==1.11",
     "pytest-sphinx==0.5.0",
     "pythreejs==2.4.2",
-    "pyvista==0.39.1",
+    "pyvista==0.40.0",
     "sphinx-autobuild==2021.3.14",
     "sphinx-autodoc-typehints==1.23.0",
     "sphinx-copybutton==0.5.2",
     "sphinx-gallery==0.13.0",
     "sphinx-notfound-page==0.8.3",
     "sphinxcontrib-websupport==1.2.4",
     "sphinxemoji==0.2.0",
+    "trame==2.5.0",
```

Comment:
@RobPasMue  Actually it has been added here (dependabot thing, not mine)

Reply:
It was @clatapie who added this.

Reply:
Use PyVista trame better

Reply:
Done then.